app/models/course.rb

```diff
@@ -37,4 +37,16 @@ def university_based?
   def further_education?
     level == 'further_education' && subjects.any? { |s| s.subject_name == 'Further education' || s.subject_code = '41' }
   end
+
+  def travel_to_work_areas
+    travel_to_work_areas = sites.map { |site| site.london_borough || site.travel_to_work_area }.uniq
+    return if travel_to_work_areas.blank?
+
+    if travel_to_work_areas.length == 1
+      travel_to_work_areas.first
+    else
+      last_travel_to_work_area = travel_to_work_areas.pop
+      travel_to_work_areas.join(', ') + " and #{last_travel_to_work_area}"
+    end
+  end
```

Comment:
Sorry i know this was my idea, but i was thinking there must be a method that does this. It turns out you can just use .to_sentence

![image](https://user-images.githubusercontent.com/42515961/107700933-16f30000-6cb0-11eb-91a0-4b54c4da500e.png)

It also behaves with an empty array 👌 

![image](https://user-images.githubusercontent.com/42515961/107701001-2ffbb100-6cb0-11eb-90dc-6e4dcff69666.png)

Reply:
Just realised that it leaves the last comma in as default. You can get around that like this.

```
["Brighton", "England", "Wales"].to_sentence
"Brighton, England, and Wales"

["Brighton", "England", "Wales"].to_sentence(last_word_connector: ' and ')
"Brighton, England and Wales"
```